provider.go

```diff
@@ -342,7 +343,11 @@ type Provider struct {
 	Read   func(context.Context, ReadRequest) (ReadResponse, error)
 	Update func(context.Context, UpdateRequest) (UpdateResponse, error)
 	Delete func(context.Context, DeleteRequest) error
-	// TODO Call
+
+	// Call allows methods to be attached to resources.
+	//
+	// Right now, Call is restricted to methods on component resources.
```

Comment:
Is the last sentence still accurate? If so, maybe give readers a hint why that is.

Reply:
I linked the associated issues in pulumi/pulumi.